backend/app/api/routes/llm.py

```
@@ -0,0 +1,34 @@
+@router.post("/llm/call")
+async def llm_call(
+    request: LLMCallRequest, _session: SessionDep, _current_user: AuthContext
+):
+    """
+    Endpoint to initiate an LLM call as a background job.
+    """
+    project_id = _current_user.project.id
+    organization_id = _current_user.organization.id
+
+    # Start background job
+    job_id = start_job(
```

Ruff (0.14.1)

Local variable `job_id` is assigned to but never used

Remove assignment to unused variable `job_id`

(F841)